File: ios/Video/RCTAirPlayButtonManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE (RCTAirPlayButtonManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(iconColor, NSNumber);
RCT_EXPORT_VIEW_PROPERTY(activeIconColor, NSNumber);
RCT_EXPORT_VIEW_PROPERTY(prioritizesVideoDevices, BOOL);

@end

File: ios/Video/RCTAirPlayButtonManager.swift
#if os(iOS)
    import React
    import UIKit

    @objc(RCTAirPlayButtonManager)
    class RCTAirPlayButtonManager: RCTViewManager {
        override func view() -> UIView {
            return RCTAirPlayButton()
        }

        override static func requiresMainQueueSetup() -> Bool {
            return true
        }
    }
#endif
